### PR TITLE
Add Fuse zkEVM testnet

### PR DESCRIPTION
**Pull request by hitesh-zeeve**

### CODE REVIEW

**Comment by hitesh-zeeve** [comment]
MR is updated

File: _data/icons/fusetestnet.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url": "ipfs://QmY3qLcSMKba5y7HTMpiWKCBCDJGXm1HNMTJ39bqH97oft",
+      "width": 200,
+      "height": 200,
+      "format": "png"
+    }
+  ]
```
Comment (macvts) [inline _data/icons/fusetestnet.json:8]: delete the last line might help, **the line after ]**

Comment (macvts) [inline _data/icons/fusetestnet.json:8]: delete all spaces before ] might help imo

**Comment by hitesh-zeeve** [comment]
All builds passed

**Review by ligi — APPROVED** [review]

**Review by ligi — APPROVED** [review]

**Pull request merged**